src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+$color-background: #cfee92;
+$color-day: #eee;
+$color-day-border: #000;
+$color-hover: pink;
+
+$size-day-cell: 100px;
```

Comment:
I think it'd be more clear if you name the variable `day-cell-size` instead of `size-day-cell`

Reply:
Thanks for your feedback, the code really looks better with these changes. Changes have been made.